go.mod

```diff
@@ -52,6 +51,7 @@ require (
 	go.opentelemetry.io/otel/metric v1.24.0 // indirect
 	go.opentelemetry.io/otel/trace v1.24.0 // indirect
 	go4.org/netipx v0.0.0-20230728184502-ec4c8b891b28 // indirect
+	golang.org/x/exp v0.0.0-20231226003508-02704c960a9b // indirect
```

Comment:
We still end up depending on this module it seems 🤷 But at least it's not a direct dependency

Reply:
Yeah, but it's not our fault at least